packages/wonder-blocks-dropdown/src/components/action-item.tsx

```diff
@@ -175,6 +173,17 @@ export default class ActionItem extends React.Component<ActionProps> {
     }
 }
 
+const theme = {
+    hover: {
+        background: semanticColor.action.filled.progressive.hover.background,
+        foreground: semanticColor.action.filled.progressive.hover.foreground,
+    },
+    press: {
+        background: semanticColor.action.filled.progressive.press.background,
+        foreground: semanticColor.action.filled.progressive.press.foreground,
+    },
```

Comment:
I would have thought that this would use the `outlined` tokens since in it's rest state, it looks more like the `outlined` styles than `filled` (it is still a bit different though)! How do we decide if we should use the `filled` or `outlined` styles?